images/federation-v2/Dockerfile

```diff
@@ -13,9 +13,10 @@
 # limitations under the License.
 
 
-FROM ubuntu:14.04
+FROM ubuntu:latest
 RUN apt-get update
 RUN apt-get install -y ca-certificates
-ADD apiserver .
 ADD controller-manager .
+ENTRYPOINT ["./controller-manager"]
+CMD ["--install-crds=false"]
```

Comment:
Why setting this as false?

Reply:
This is because the CRDs are created by the `install.yaml` config and likewise can be deleted via the same config. It also provides more flexibility e.g. don't need to support all CRDs if you don't care about them all.

Reply:
Kubebuilder generates this flag and sets it to true by default. We prefer to deploy the CRDs outside the controller.